Cargo.toml

```diff
@@ -43,7 +43,7 @@ license = "MIT"
 repository = "https://github.com/denoland/deno"
 
 [workspace.dependencies]
-deno_ast = { version = "=0.36.2", features = ["transpiling"] }
+deno_ast = { version = "0.37.0", features = ["transpiling"] }
```

Comment:
An example of how unpinning could break `cargo install deno` is the 0.37.1 release of deno_ast.